core/tests/build_sources/assets/constants.js

```diff
@@ -446,6 +446,8 @@ var constants = {
   "ACTIVITY_STATUS_PRIVATE": "private",
   "ACTIVITY_STATUS_PUBLIC": "public",
 
+  "SITE_FEEDBACK_FORM_URL": "",
```

Comment:
When is this actually populated?

Reply:
See https://github.com/oppia/oppia/tree/develop/core/tests/build_sources -- this is just for production testing. Maybe we should include a dummy URL in it.

Reply:
I noticed that there's actually a URL attached to it on the 404 page

Reply:
@seanlip , should I add the dummy URL and commit in this pull request?

Reply:
Yeah, I think that's OK.